src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -152,6 +158,22 @@
       <Name>System.Globalization</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.7'">
```

Comment:
>ItemGroup [](start = 3, length = 9)

would it be clearer to move all these tests into system.runtime.tests, since the types are there now?